api/src/main/java/org/apache/cloudstack/api/command/user/iso/ListIsosCmd.java

```diff
@@ -88,6 +90,11 @@ public class ListIsosCmd extends BaseListTaggedResourcesCmd implements UserCmd {
     @Parameter(name = ApiConstants.SHOW_RESOURCE_ICON, type = CommandType.BOOLEAN, description = "flag to display the resource image for the isos")
     private Boolean showIcon;
 
+    @Parameter(name = ApiConstants.ARCHITECTURE, type = CommandType.STRING,
+            description = "the CPU architecture of the ISO. Valid options are: x86_64, arm64",
```

Comment:
same comments as for templates. Btw on second though, this is fine too as long as we stick to it.